### DISCUSSION TITLE
Are there real world examples of do-mpc use case?

### DISCUSSION THREAD

**Discussion by edumagol** [do-mpc/do-mpc#276] [Show and tell]
Hi guys, amazing work and repo! Probably one of the few in the world of mpc implemented in python!

Would like to know if you guys can give us other examples of real world scenarios where do-mpc was used/implemented?

Really interested in developing a product using do-mpc as a backend.

All the best,
Edu

**Comment by 4flixt** [comment]
Dear Edu, there are some and I'll try add more later but here is the most recent work using do-mpc: https://ieeexplore.ieee.org/document/9797728

**Reply by edumagol** [reply]
Hi @4flixt , thanks for the amazing support and sharing this paper! Really good one, by the way! If you find more, just let me know.

All the best,
Edu
